### ISSUE TITLE
ActionPopoverMenuButton has a default aria-label even with a visible text

### ISSUE THREAD

**Issue by ernon66** [Sage/carbon#7143]
### Description

A default `aria-label` is added to `ActionPopoverMenuButton` even if there is already a discernible text, which is not recommended and causes screen readers to only read the `aria-label` and ignore the carbon user defined text

### Reproduction

https://stackblitz.com/edit/parsium-carbon-starter-suxhour4?file=src%2FApp.tsx,src%2FApp.spec.tsx,package.json

### Steps to reproduce

In Stackblitz, the unit test trying to get a button with "Actions" name fails because aria-label already has a default "actions" value

If an `ActionPopoverMenu` button with a different discernible text is rendered, a screen reader will read "actions" anyway

### Carbon version

146.2.1

### Relevant browsers

Chrome

### Relevant OSs

MacOS

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by sianford** [comment]
Hi @ernon66. There are some instructions in the docs in the Storybook library for this scenario. If you navigate to the [Custom Menu Button story](https://carbon.sage.com/iframe.html?viewMode=docs&id=action-popover--docs&globals=&args=#with-custom-menu-button), it recommends that "set this to undefined if your button contains visible text, as this will prevent screen readers from reading the visible label." Hope that helps :)

**Comment by nuria1110** [comment]
Hi @ernon66, thank you for the update - looks like you are setting the aria-label on ActionPopover, instead it should be set on the ActionPopoverMenuButton: https://stackblitz.com/edit/parsium-carbon-starter-ocjfycet?file=src%2FApp.tsx
